scripts/asr_language_modeling/ngram_lm/train_kenlm.py

```diff
@@ -44,6 +44,7 @@
 import torch
 
 import nemo.collections.asr as nemo_asr
+from nemo.collections.asr.parts.submodules.ctc_beam_decoding import DEFAULT_TOKEN_OFFSET
```

Comment:
If you import the DEFAULT_TOKEN_OFFSET, then why not use DEFAULT_TOKEN_OFFSET instead of TOKEN_OFFSET in the code?
Other way is to use the name TOKEN_OFFSET instead of DEFAULT_TOKEN_OFFSET?

Reply:
I left DEFAULT_TOKEN_OFFSET only